### DISCUSSION TITLE
Tracking MD simulation

### DISCUSSION THREAD

**Discussion by SamFreemanFox** [BradyAJohnston/MolecularNodes#633] [Q&A] [answered]
Love the plugin - just wondered whether there is a simple way to get the camera to track a protein during the MD trajectory? I have a protein which moves during the trajectory  but would like to keep it centred in frame. I have tried the "track to" constraint but to no avail. 

Thanks!

**Comment by BradyAJohnston** [comment, marked as answer]
There are ways to track a camera to a group of vertices, but with Geometry Nodes it becomes a bit tricky as the systems don't quite talk to each other super well currently. 

There is a node [`Centre on Selection`](https://bradyajohnston.github.io/MolecularNodes/nodes/geometry.html#centre-on-selection) in Molecular Nodes though that will centre whatever is selected on the world origin. If you use that and select the protein (or the part of the protein that you are interested in) then you can point the camera in the same place the whole time.

**Reply by SamFreemanFox** [reply]
Thanks Brady, that's definitely worked better than expected
